rooms.js

```diff
@@ -1350,7 +1350,8 @@ class ChatRoom extends Room {
 	}
 	getIntroMessage(user) {
 		let message = '';
-		if (this.introMessage) message += '\n|raw|<div class="infobox infobox-roomintro"><div' + (!this.isOfficial ? ' class="infobox-limited"' : '') + '>' + this.introMessage.replace(/\n/g, '') + '</div>';
+		let scroll = Db.DisabledScrolls.get(this.id, false);
+		if (this.introMessage) message += '\n|raw|<div class="infobox infobox-roomintro"><div' + ((!this.isOfficial || !scroll) ? ' class="infobox-limited"' : '') + '>' + this.introMessage.replace(/\n/g, '') + '</div>';
```

Comment:
Scroll won't show up,if the room has scroll disabled or if it is official isn't it?